worker-management-component/src/main/java/shop/wazard/adapter/out/persistence/WorkerManagementDbAdapter.java

```diff
@@ -59,12 +56,8 @@ public AccountForWorkerManagement findAccountByEmail(String email) {
 
     @Override
     public void updateWaitingStatus(WaitingInfo waitingInfo) {
-        AccountJpa accountJpa = accountForWorkerManagementRepository.findById(waitingInfo.getAccountId())
-                .orElseThrow(() -> new AccountNotFoundException(StatusEnum.ACCOUNT_NOT_FOUND.getMessage()));
-        CompanyJpa companyJpa = companyForWorkerManagementRepository.findById(waitingInfo.getCompanyId())
-                .orElseThrow(() -> new CompanyNotFoundException(StatusEnum.COMPANY_NOT_FOUND.getMessage()));
-        WaitingListJpa waitingListJpa = waitingListForWorkerManagementRepository.findByAccountJpaAndCompanyJpa(accountJpa, companyJpa)
+        WaitingListJpa waitingListJpa = waitingListForWorkerManagementRepository.findById(waitingInfo.getWaitingListId())
                 .orElseThrow(() -> new WorkerNotFoundInWaitingListException(StatusEnum.WORKER_NOT_FOUND_IN_WAITING_LIST.getMessage()));
-        waitingListJpa.updateWaitingStatus();
+        waitingListJpa.updateWaitingStatus(WaitingStatusJpa.valueOf(waitingInfo.getWaitingStatus().getStatus()));
```

Comment:
이걸 mapper로 한번 더 감싸줄래
```suggestion
맵퍼.updateWaitingStatus(waitingListJpa, waitingInfo);

// 맵퍼 내부코드
public void updateWaitingStatus(waitinglistjpa, waitinginfo){
  waitingListJPa.updateWaitingStatus(waitinginfo);
}
```

매개변수로 status 꺼내서 주는것보다 더 깔끔해 보임